File: Data Structures & Algorithms/concatenation-of-array/submission-0.cpp
class Solution {
public:
    vector<int> getConcatenation(vector<int>& nums) {
        vector<int> res;

        for (int j = 0; j < 2; j++) {
            for (int i = 0; i < nums.size(); i++) {
                res.push_back(nums[i]);
            }
        }

        return res;
    }
};

File: Data Structures & Algorithms/maximum-subarray/submission-0.py
class Solution:
    def maxSubArray(self, nums: List[int]) -> int:
        n = len(nums)
        curr = nums[0]
        res = nums[0]
        for n in nums[1:]:
            curr = max(n, curr + n)
            res = max(res, curr)
        return res